pkg/clioptions/clusterdiscovery/provider.go

```diff
@@ -28,6 +28,9 @@ import (
 	// Initialize nutanix as a provider
 	_ "github.com/openshift/origin/test/extended/util/nutanix"
 
+	// Initialize nutanix as a provider
```

Comment:
This comment should say external instead of nutanix

Reply:
opened a dedicated PR https://github.com/openshift/origin/pull/29737/files